articles/understanding-dbms/index.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/understanding-dbms/
+title: Understanding Database Architecture and Models
+description: Learn the concepts of database and its three types of build architecture. Study the different aspects of the Hierarchical Data model, Network Data model and Relational Data model.
+author: kanishkvardhan-a-n
+date: 2020-09-21T00:00:00-05:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/understanding-dbms/hero.jpg
+    alt: Database Architecture and Models example image
+
+---
+Computers have made our lives easier in handling complicated calculations and solving complex problems. Hands down computers are one of the best inventions of mankind. While using computers and instilling them in our lives has made our work a piece of cake, maintaining them can be a difficult to accomplish.

```

Comment:
remove "a" from the last sentence section "maintaining them can be a difficult to accomplish"